scope.json

```diff
@@ -202,5 +202,30 @@
         "alimama.com"
       ]
     }
+{
```

Comment:
Can you make sure to add `,` before this line as this breaks the JSON structure?

Reply:
changes made